src/hooks/useKeyboardNavigation.js

```diff
@@ -0,0 +1,131 @@
+import PropTypes from 'prop-types';
+import { useEffect } from 'react';
+import tinykeys from 'tinykeys';
+
+const SEARCH_SECTION_INDEX = 1;
+
+const useKeyboardNavigation = ({
+  categoriesNavRef,
+  emojiSearchRef,
+  emojiListRef,
+}) => {
+  useEffect(() => {
+    return tinykeys(categoriesNavRef.current, {
+      ArrowLeft: focusPrevCategory,
+      ArrowRight: focusNextCategory,
+      ArrowDown: focusNextSection,
+    });
+  }, []);
+
+  useEffect(() => {
+    if (!emojiSearchRef.current) return;
+
+    insertSearchSection();
+
+    return tinykeys(emojiSearchRef.current, {
+      ArrowRight: focusSkinTonePicker,
+      ArrowUp: focusPrevSection,
+      ArrowDown: focusNextSection,
+    });
+  }, []);
+
+  useEffect(() => {
+    return tinykeys(emojiListRef.current, {
+      ArrowUp: focusPrevSection,
```

Comment:
This is partially correct.
ArrowUp needs to
1. If we're on the first row, select the previous section
2. If we're on any other row, focus the above emoji (requires some math, can be incremental).

+ Similar behavior for ArrowDown.

Let's not implement it now, and instead, as long as we don't support it, let's not add the arrowUp binding.
```suggestion
```